src/clearskies/cursors/cursor.py

```diff
@@ -149,8 +155,11 @@ def execute(self, sql: str, parameters: tuple | list):
         -------
             Result of cursor.execute().
         """
-        print(f"Executing SQL: {sql} with parameters: {parameters}")
-        return self.cursor.execute(sql, parameters)
+        try:
+            return self.cursor.execute(sql, parameters)
+        except Exception as e:
+            self.logger.exception(f"Error executing SQL: {sql} with parameters: {parameters}")
```

Comment:
Make sure and include the error message from the exception in the error message.  At the end, maybe just put in `with parameters: {parameters}.  SQL error: {str(e)}"`

Reply:
That is not needed, the stack of the original error back too